.github/workflows/cve-scanning.yml

```diff
@@ -25,9 +25,9 @@ jobs:
       matrix:
         node-version: [20]
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
```

Comment:
Might be worth bumping to 4.1.7 / 692973e3d937129bcbf40652eb9f2f61becf3332 (not tested - but used in the package workflow) to be up to date.